tests/ManageCourses.Tests/SmokeTests/SystemUnderTest.cs

```diff
@@ -66,6 +65,71 @@ public void DataExportWithEmptyCampus()
             Assert.AreEqual("Main campus site", campus.Name, "ProviderCourses.CourseDetails.Variants.Campuses.Name should be retrieved");
         }
 
+        [Test]
+        public void DataExport_badAccesCode_404()
```

Comment:
I think the tests for testing the api key auth should live in a different test class even though they test the same endpoint.

Reply:
hmm partial class?


Reasons stills stand for system under test
* It is testing the exact same endpoint twice using the exact same configuration variated by its input
* both input are valid for their respective authentication mechanism.
* system is configured for only one of the authentication mechanism.

Reply:
see above comment thread.